Jenkinsfile

```diff
@@ -698,6 +698,17 @@ core_logic: {
         }
       }
     },
+    'Scala: CentOS CPU': {
+      node(NODE_LINUX_CPU) {
+        ws('workspace/ut-scala-centos7-cpu') {
+          timeout(time: max_time, unit: 'MINUTES') {
+            utils.unpack_and_init('centos7_cpu', mx_lib, true)
```

Comment:
mx_lib should be mx_dist_lib. Also, change mx_lib to mx_dist_lib on line 155 as well. mx_dist_lib packs and unpacks more files which are necessary for scala build.

Reply:
What diff on the files that mx_dist_lib will provide?

Reply:
It adds 3rdparty/ps-lite/build/libps.a, deps/lib/libprotobuf-lite.a, and deps/lib/libzmq.a. According to the comment at the top, they are necessary for scala with dist_kvstore